tests/sentry/tasks/test_post_process.py

```diff
@@ -1643,6 +1644,56 @@ def test_sdk_crash_monitoring_is_not_called_without_project_id(self, mock_sdk_cr
         mock_sdk_crash_detection.detect_sdk_crash.assert_not_called()
 
 
+@mock.patch("sentry.utils.metrics.incr")
+class ReplayLinkageTestMixin(BasePostProgressGroupMixin):
+    def test_replay_linkage(self, incr):
+        replay_id = uuid.uuid4().hex
+        event = self.create_event(
+            data={"message": "testing", "contexts": {"replay": {"replay_id": replay_id}}},
+            project_id=self.project.id,
+        )
+
+        with self.feature({"organizations:session-replay-event-linking": True}):
+            self.call_post_process_group(
+                is_new=True,
+                is_regression=False,
+                is_new_group_environment=True,
+                event=event,
+            )
+            incr.assert_called_with("post_process.process_replay_link.id_exists")
```

Comment:
id_sampled is also incremented.  Not that it matters too much.  Just thought I'd mention.

Reply:
true! will add an assertion